extension/observer/kafkatopicsobserver/extension_test.go

```diff
@@ -105,11 +105,11 @@ func TestCollectEndpointsAllConfigSettings(t *testing.T) {
 			Details: &observer.KafkaTopic{},
 		},
 	}
-	require.Equal(t, want, kEndpoints)
+	require.True(t, endpointsEqual(want, kEndpoints))
```

Comment:
You can simply use `require.ElementsMatch(t, want, kEndpoints)`, same for below

Reply:
Done